### ISSUE TITLE
Element integration with Commerce Products

### ISSUE THREAD

**Issue by AndrewMac** [verbb/formie#768]
**What are you trying to do?**
We have a client who is looking at setting up a Craft Commerce "marketplace", where users could add products they want to sell through the site. The client doesn't want to give those users access to the control panel, so we need to let them add/edit their products through forms on the front end. It would be absolutely brilliant if Formie could let them do that!

**What's your proposed solution?**
Formie! ;-)

**Additional context**
From previous discussions we've had, I know that integration with Products is tricky because a product can have multiple variants. But in this case we don't really see any need for anything other than the default variant. So it would certainly be okay for us if a Formie solution was limited to just a single default variant. It may not be perfect for everyone, but it would be a start ... and i's still better than nothing ;-)

**Comment by engram-design** [comment]
Yep great idea (originally from https://github.com/verbb/formie/issues/160) and there's some things we'll need to think about, such as dealing with multiple variants. But maybe to keep it simple for the moment, we go with single-variants - as you say.

**Comment by AndrewMac** [comment]
Hi Josh, thanks so much for considering this request. I don't honestly know if I have the skills required to implement this kind of thing without support from something like Formie, so I very much appreciate your willingness to think about it. And I'm sure that in the longer term it wouldn't be just me that would get enormous value from this feature, if implemented.

**Comment by engram-design** [comment]
Updated in [2.2.0](https://github.com/verbb/formie/releases/tag/2.2.0) and [3.1.0](https://github.com/verbb/formie/releases/tag/3.1.0).
